packages/harmony/src/components/button/FollowButton/types.ts

```diff
@@ -27,4 +40,9 @@ export type FollowButtonProps = {
    * Callback for when an unfollow is triggered.
    */
   onUnfollow?: () => void
-} & Omit<HTMLProps<HTMLInputElement>, 'size'>
+
+  /**
+   * Html input type to use
+   */
+  type?: 'button' | 'checkbox'
```

Comment:
this is the problem here i think, type should not be in the "base props", just in the inputprops